public/app/ui/pages/about/AboutPage.js

```diff
@@ -7,7 +7,7 @@
         ],
         function (container) {
             return function () {
-                this.ready = function (callback) {
+                this.ready = function (element, callback) {
```

Comment:
Bugfix, parameters were incorrect.  This one was copied and pasted to a bunch of different files so it's been fixed in all of them.